<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提示：忘记密码，可通过验证信息请求密码重置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>图书管理系统</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>114</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>实作者：游朝江</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>班级：11计科A1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_test">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/logininback.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_getpicture">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>获取头像</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_login">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(13, 109, 193);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(13, 109, 193);</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>92</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>卡号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>145</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注册用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_qrcorelogin">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>142</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>二维码登陆</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_logininte">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_user">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_passwd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWebView" name="webView">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QWebView" name="webView_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <zorder>groupBox_logininte</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_test</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_getpicture</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_qrcorelogin</zorder>
  <zorder>webView</zorder>
  <zorder>webView_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
